### ISSUE TITLE
iPhone meeting requests show as ICS attachments

### ISSUE THREAD

**Issue by sirvulcan** [fmbiete/Z-Push-contrib#234]
When I receive a meeting request on my iPhone it shows up as an ICS attachment. I can then go into the attachment and accept it. I am not sure if this is intentional or not. On Outlook and on Android the meeting requests show up in the initial email.

My iPhone is currently running iOS 8.3. Let me know if you need any further details.

![img_0413](https://cloud.githubusercontent.com/assets/1088408/10179868/74a867ea-6765-11e5-9ed0-1dc45ef5ee76.PNG)
![img_0414](https://cloud.githubusercontent.com/assets/1088408/10179869/74b419b4-6765-11e5-964a-a8c3bb496d8a.PNG)

**Comment by fmbiete** [comment]
Yes, it's intended. Or rather a consequence of iPhone requesting the original MIME messages. They don't request the prepared event but the original message where the event is an attached file.

Could you test the behaviour against a real Exchange server? Maybe it's different....
